sql/core/src/test/resources/sql-tests/inputs/pgSQL/int4.sql

```diff
@@ -33,6 +33,10 @@ INSERT INTO INT4_TBL VALUES ('-2147483647');
 -- INSERT INTO INT4_TBL(f1) VALUES ('123       5');
 -- INSERT INTO INT4_TBL(f1) VALUES ('');
 
+-- We cannot test this when checkOverflow=true here
+-- because exception happens in the executors and the
+-- output stacktrace cannot have an exact match
+set spark.sql.arithmetic.checkOverflow=false;
```

Comment:
is it necessary? It's false by default.

Reply:
I put it for enforcing this so that eventual changes on the default value won't affect here.